template/src/@types/react-navigation.d.ts

```diff
@@ -1,8 +1,8 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-import type {RootStackParamList} from '@navigation/navigators/RootStackNavigator';
+import type { RootStackParamList } from '@navigation/navigators/RootStackNavigator'
 
 declare global {
   namespace ReactNavigation {
-    interface RootParamList extends RootStackParamList {}
+    type RootParamList = RootStackParamList
```

Comment:
Why this `interface` -> `type` change?

Reply:
because of rule: 
`ESLint: An interface declaring no members is equivalent to its supertype.(@typescript-eslint/no-empty-interface)`